content/patterns/button/examples/button.html

```diff
@@ -195,6 +195,18 @@ <h2 id="sc1_label">HTML Source Code</h2>
           sourceCode.make();
         </script>
       </section>
+
+
+      <section>
+        <h2>Support Levels</h2>
+        <iframe
+          class="support-levels-command-button"
+          src="https://aria-at-app-sandbox.bocoup.com/embed/reports/command-button"
+          height="500"
+          allow="clipboard-write"
+          style="border-style: none; width: 100%;">
+        </iframe>
+      </section>
```

Comment:
Need an h3 for "Toggle Button" followed by another iframe for that table.

Reply:
Toggle Button has been added